src/styles/main.scss

```
@@ -1,3 +1,53 @@
+$heigth: 300px;
+$colors:
+  1 rgb(255, 0, 0),
+  2 rgb(204, 0, 0),
+  3 rgb(153, 0, 0),
+  4 rgb(102, 0, 0),
+  5 rgb(51, 0, 0),
+  6 rgb(0, 0, 0);
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+}
+
 body {
   margin: 0;
 }
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(1,minmax(300px,600px));
```

```suggestion
  grid-template-columns: repeat(auto-fill, minmax(300px,1fr));
```
Мне кажется лучше не ограничивать количество колонок и максимальную ширину блока 
Вот такая запись даст возможность не ограничивать колонки и ширину блоков. Так же она даст возможность сократить код ниже

---

Спасибо!